django/db/models/lookups.py

```diff
@@ -151,7 +151,7 @@ def process_lhs(self, compiler, connection, lhs=None):
         lhs_sql = connection.ops.field_cast_sql(
             db_type, field_internal_type) % lhs_sql
         lhs_sql = connection.ops.lookup_cast(self.lookup_name, field_internal_type) % lhs_sql
-        return lhs_sql, params
+        return lhs_sql, list(params)
```

Comment:
Well, the return value is supposed to be a list since `.extend()` is called on it later on. The documentation, gives the example of a tuple, and this is broken as well (Something like `AttributeError: tuple doesn't have any attribute extend`).

So I do feel `list()` is well suited here. If you don't think so, I can always fix that.

Just let me know your thoughts, thanks.